test/Transforms/PagerandoWrappers/varargs.ll

```diff
@@ -0,0 +1,31 @@
+; RUN: opt < %s -pagerando-wrappers -S | FileCheck %s
+
+declare void @llvm.va_start(i8*)
+declare void @llvm.va_end(i8*)
+
+define i32 @varags(...) {
```

Comment:
This is a good start, but we should cover some other, more complex varargs cases such as va_copy and va_start in optional branches. I can write those up if you want.

Reply:
Noted. Will add more complex cases.